bin/crew

```diff
@@ -854,15 +854,13 @@ def build_and_preconfigure(target_dir)
   Dir.chdir target_dir do
     puts 'Building from source, this may take a while...'
 
-    if CREW_LA_RENAME_ENABLED
-      # Rename *.la files temporarily to *.la_tmp to avoid
-      # libtool: link: '*.la' is not a valid libtool archive.
-      # See https://gnunet.org/faq-la-files and
-      # https://stackoverflow.com/questions/42963653/libquadmath-la-is-not-a-valid-libtool-archive-when-configuring-openmpi-with-g
-      puts 'Rename all *.la files to *.la_tmp'.lightblue
```

Comment:
This clobbers the `CREW_LA_RENAME_ENABLED` logic. Please rebase against current master?